client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -113,7 +113,7 @@ const CartItem = React.createClass( {
 		} else if ( getIncludedDomain( this.props.cartItem ) ) {
 			info = getIncludedDomain( this.props.cartItem );
 		} else if ( isTheme( this.props.cartItem ) ) {
-			info = this.props.selectedSite.domain;
+			info = this.props.selectedSite && this.props.selectedSite.domain;
```

Comment:
How I read it is:

If it's a theme, there's no `meta`. And if there's no `meta` it falls back to the same value we have here. Maybe I'm wrong. :)